crates/stages/src/stages/headers.rs

```diff
@@ -92,6 +95,9 @@ impl<DB: Database, D: HeaderDownloader, C: Consensus, H: HeadersClient, S: Statu
         while let Some(headers) = stream.next().await {
             match headers.into_iter().collect::<Result<Vec<_>, _>>() {
                 Ok(res) => {
+                    // Register elapsed time of request
+                    histogram!("stages.headers.request_time", start.elapsed());
```

Comment:
Should probably be in the downloader itself since we might replace this with a concurrent downloader (and in that case this metric is wrong)